c10/util/flat_hash_map.h

```diff
@@ -190,21 +191,6 @@ inline int8_t log2(uint64_t value) {
   return table[((value - (value >> 1)) * 0x07EDD5E59A4E28C2) >> 58];
 }
 
-template <typename T, bool>
```

Comment:
This whole file is taken from another repo (see top of the file). i'm not sure we want to lint it in case we need to get updates from it from the repo in the future.

Reply:
@albanD There are local changes recorded in the top of the file. I prefer to keep the current changes and exclude it from scan.